diff --git a/dist/concurrency/built-in/Browser.js b/dist/concurrency/built-in/Browser.js
index 10d7d66dc4d5ad4a63ff27f0f3bf44ef754e44d3..8d22c1e9976efe46cad086bfc0d38329216492cc 100644
--- a/dist/concurrency/built-in/Browser.js
+++ b/dist/concurrency/built-in/Browser.js
@@ -29,7 +29,7 @@ class Browser extends ConcurrencyImplementation_1.default {
             return {
                 jobInstance: () => __awaiter(this, void 0, void 0, function* () {
                     yield (0, util_1.timeoutExecute)(BROWSER_TIMEOUT, (() => __awaiter(this, void 0, void 0, function* () {
-                        context = yield chrome.createIncognitoBrowserContext();
+                        context = yield chrome.createBrowserContext();
                         page = yield context.newPage();
                     }))());
                     return {
diff --git a/dist/concurrency/built-in/Context.js b/dist/concurrency/built-in/Context.js
index 7332024a77324a1d0ff8f88b4d220f55525938a5..86a969ed7e4793c762b40d804bb8ee4d5fe13398 100644
--- a/dist/concurrency/built-in/Context.js
+++ b/dist/concurrency/built-in/Context.js
@@ -14,7 +14,7 @@ class Context extends SingleBrowserImplementation_1.default {
     createResources() {
         return __awaiter(this, void 0, void 0, function* () {
             const context = yield this.browser
-                .createIncognitoBrowserContext();
+                .createBrowserContext();
             const page = yield context.newPage();
             return {
                 context,
